cf7-pf_oli_klay/templates/qr_bill.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
```

Comment:
I thought we wouldn't use python, what is this file for?

Reply:
Yeah my bad I've commited a file that I was using for testing